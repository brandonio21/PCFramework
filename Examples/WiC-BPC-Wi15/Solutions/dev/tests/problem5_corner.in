5 7 2 3
3
3
5
2 1 2 3
2
5 5 3 3 1
3
2 3 13 9 5 7 4 2 1 1 5 10 50 3 8 9
5
1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19
19
1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19
39
9 0 48 38 992 7 38 9
0
6
1
7
5
6 6 6 6 6 6 6 6 6 6
4
6 6 6 6 6 6 6 6 7
4
0 0 0 0 0
8
