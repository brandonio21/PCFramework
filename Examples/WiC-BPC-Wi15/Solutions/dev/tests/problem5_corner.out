15
3
5
13
91
190
190
0
6
7
24
25
0
